niftypad/api/__init__.py

```diff
@@ -17,6 +17,21 @@ def kinetic_model(src, dst=None, params=None, model='srtmb_basis', w=None, r1=0.
       params (Path or str): config (relative to `src` directory)
       model (str): srtmb_basis, srtmb_k2p_basis, srtmb_asl_basis, logan_ref, logan_ref_k2p,
         mrtm, mrtm_k2p
+      # more args descriptions
+      input_interp_method (str): linear, cubic, exp_1, exp_2, feng_srtm
+      (the interpolation method for getting reference input)
+      w (None or numpy array): weights for weighted model fitting
+      r1 (a value): a pre-chosen value between 0 and 1 for r1, used in srtmb_asl_basis
+      k2p (a value): a pre-chosen value for k2p, in second^-1, used in srtmb_k2p_basis, logan_ref_k2p, mrtm_k2p
+      beta_lim (a list): [beta_min, beta_max] for setting the lower and upper limits of beta values in basis functions,
+       used in srtmb_basis, srtmb_k2p_basis, srtmb_asl_basis
+      n_beta (integer): number of beta values / basis functions, used in srtmb_basis, srtmb_k2p_basis, srtmb_asl_basis
+      linear_phase_start (integer): the second when the linear phase starts, used in logan_ref, logan_ref_k2p, mrtm, mrtm_k2p
```

Comment:
@JJiao You mean, "start time of linear phase in seconds" correct?

Reply:
Yes you are right.